### ISSUE TITLE
Cannot compile dateformatter

### ISSUE THREAD

**Issue by nkovacs** [globalizejs/globalize#722]
With these dependencies:
```
"cldr-data": "^30.0.0",
"globalize": "^1.3.0-alpha.3",
"globalize-compiler": "^0.3.0-alpha.3"
```
and this code:

```
var Globalize = require( "globalize" );
Globalize.load( require( "cldr-data" ).entireSupplemental() );
Globalize.load( require( "cldr-data" ).entireMainFor( "en", "ar-AE", "ar-SA" ) );

require('globalize-compiler').compile([
    Globalize( "en" ).dateFormatter({ skeleton: "GyMMMd" })
], {});
```

I get Error: No formatters or parsers has been provided.
This works in 1.2.0.

The compiler test in #721 also fails because of this.

**Comment by nkovacs** [comment]
Fixed by ee74907c15fc86be47ee21701520d9fb914b4822, thanks.

**Comment by rxaviers** [comment]
Thanks for the note and for closing.
